packages/ui-preset/src/theme/extension/theme.ts

```diff
@@ -1,283 +1,286 @@
-export const theme = {
-  "extend": {
-    "colors": {
-      "ui": {
-        "code": {
-          "text": {
-            "base": {
-              "DEFAULT": "var(--code-text-base)"
-            },
-            "subtle": {
-              "DEFAULT": "var(--code-text-subtle)"
-            }
-          },
-          "bg": {
-            "base": {
-              "DEFAULT": "var(--code-bg-base)"
-            },
-            "header": {
-              "DEFAULT": "var(--code-bg-header)"
-            }
-          },
-          "border": {
-            "base": {
-              "DEFAULT": "var(--code-border-base)"
-            },
-            "strong": {
-              "DEFAULT": "var(--code-border-strong)"
-            }
-          },
-          "icon": {
-            "DEFAULT": "var(--code-icon)"
-          }
+import { ThemeConfig } from "tailwindcss/types/config"
+
+export const theme: Partial<ThemeConfig> = {
```

Comment:
This file is auto generated, so all of these changes will be overwritten. Ideally we should pull these text tokens from Figma using the toolbox CLI, but since the tokens are still WIP this should at least be moved to a separate file for now :)

Reply:
Moved them to a dumb file. It can definitely be structured/typed better, but as it's super temporary I wouldn't bother?